articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
 
-* [ament_package](https://github.com/ament/ament_package) is a Python package providing an API to parse the files containing the meta information.
-* [ament_cmake](https://github.com/ament/ament_cmake) contains a set of packages which provide CMake based functionality.
-  The features are split across several packages in order to ensure that they are cleanly separated from each other.
-  The main feature of the CMake packages is to automatically generate a CMake config file to pass information to downstream packages.
-* [ament_lint](https://github.com/ament/ament_lint) contains a set of packages which perform linting tasks.
-  For a set of linters a ROS 2 specific configuration is provided which can be used via the command line as well as from within CMake (commonly as a CTest).
-* [ament_tools](https://github.com/ament/ament_tools) is a Python package which provides command line tools to build, test and install packages.
-  It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
-  Each different build system is integrated through extension points.
-  While it currently does not build packages in parallel that will be added in the future to speed up the build process.
-* [ament_index](https://github.com/ament/ament_index) contains a set of packages which provide API to access the [ament resource index](https://github.com/ament/ament_cmake/blob/master/ament_cmake_core/doc/resource_index.md).
-  Currently it only contains a Python implementation but other languages like C++ should follow.
+Each package is build separately with its own build system.
+In order to make the output of one package available to other packages each package can extend the environment in a way that downstream packages can find and use its artifacts and resources.
+If the resulting artifacts are installed into e.g. `/usr` it might not be necessary to alter the environment at all since these folders are commonly being searched by various tools.
+
+
+### The ament command line tool
+
+[ament_tools](https://github.com/ament/ament_tools) is a Python package which provides the command line tool `ament` to build, test, install, and uninstall packages.
+It is similar to [catkin_tools](https://github.com/catkin/catkin_tools) and builds each package in a workspace in topological order.
+Support for different build systems is integrated through extension points.
+
+While it currently does not build packages in parallel that feature will be added in the future to speed up the build process.
+The goal is to reuse common functionality from catkin_tools by making it available through a third package which can be used by both tools.
 
 
 ### Integrate arbitrary build systems
 
-Each package can utilize a different build system to perform the steps of configuring, building and installing.
+Each package can utilize a different build system to perform the steps of configuring, building, and installing.
```

Comment:
We don't actually have a configuring step, not sure if that matters in this context.